src/js/user-profile/components/FormItem.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import moment from 'moment';
+
+import { dateDiffDesc } from '../../common/utils/helpers';
+import { formTitles, formLinks } from '../helpers';
+
+class FormItem extends React.Component {
+  render() {
+    const savedFormData = this.props.savedFormData;
+    const formId = savedFormData.form;
+    const { last_updated: lastSaved, expires_at: expirationTime } = savedFormData.metadata;
+    const lastSavedDateTime = moment.unix(lastSaved).format('M/D/YYYY [at] h:mm a');
+    const expirationDate = moment.unix(expirationTime);
+    const isExpired = moment(expirationDate).isBefore();
+    const activeView = (
+      <div className="card information">
+        <div className="row small-collapse saved-form-information">
+          <div className="small-12 large-8 columns saved-form-metadata-container">
+            <h5 className="saved-form-title">Application for {formTitles[formId]}</h5>
+            {!!lastSaved && !!expirationDate && <div>
+              <span className="saved-form-item-metadata">Last saved on {lastSavedDateTime}</span>
+              <span className="saved-form-item-metadata saved-form-item-expires">Your saved application will expire in {dateDiffDesc(expirationDate)}.</span>
+            </div>}
+          </div>
+          <div className="small-12 large-4 columns resume-saved-application-container">
+            <a className="usa-button-primary resume-saved-application" href={formLinks[formId]}>Continue Your Application</a>
+          </div>
+        </div>
+        <div className="remove-saved-application-container">
+          <button className="va-button-link" onClick={() => {this.props.toggleModal(formId);}}>
+            <i className="fa fa-trash"></i><span className="remove-saved-application-label">Delete</span>
+          </button>
+        </div>
+      </div>
+    );
+    const expiredView = (
```

Comment:
Doesn't really affect your changes, but this view is really only ever going to be visible for an hour, where the form is expired but before the scheduled job removes it.

Reply:
Thanks @jbalboni, does it make sense to inform them of this should this wish to retrieve information from their saved form? @stacyhlinn @peggygannon could you please share your thoughts on this?

Reply:
I don't think we can do much to change any of this right now, the backend would have to work differently.

Reply:
@jbalboni We should somehow notify the user that their application has expired, either by keeping the yellow card, or including in the notification message that we're exploring separately.